Makefile

```diff
@@ -25,7 +25,7 @@ resettestdb:
 	FLASK_ENV=test make resetdb
 
 resetdb:
-	python3.7 -m pipenv run python resetdb.py
+	FLASK_ENV=$${FLASK_ENV:-development} python3.7 -m pipenv run python resetdb.py
```

Comment:
cool, didn't know about this syntax

Reply:
It's a mashup of `make` and `bash`. `${FOO:-bar}` is a bash-ism which expands to the value of `FOO` if it is set, but defaults to `bar` if not. Prefixing that with `$` ensures `make` doesn't try to treat it as a `make` expansion.